examples/evaluation/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,1090 @@
+            ]
+          },
+          "execution_count": 9,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "response_vector.source_nodes"
```

Comment:
This is a bit much to dump on the reader. Either pick a few key fields, or if you think all are important, then split it up into blocks with a line or two directing the reader's attention.

Reply:
Makes sense. Will update.